packages/dids/tests/methods/did-jwk.spec.ts

```diff
@@ -184,12 +186,7 @@ describe('DidJwk', () => {
         ],
       };
 
-      try {
-        await DidJwk.getSigningMethod({ didDocument });
-        expect.fail('Error should have been thrown');
-      } catch (error: any) {
-        expect(error.message).to.equal('Method not supported: example');
-      }
+      await expect(DidJwk.getSigningMethod({ didDocument })).to.eventually.be.rejectedWith('Method not supported: example');
```

Comment:
revert to using try / catch and not chaiAsPromised
```suggestion
      try {
        await DidJwk.getSigningMethod({ didDocument });
        expect.fail('Error should have been thrown');
      } catch (error: any) {
        expect(error.message).to.equal('Method not supported: example');
      }
```